jest/fallback.env.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
As noted in your comment in the `jest.config.js` file, I'll very likely remove this for the PR because I agree that `frontend-build` should already provide this fallback.